inference-engine/tests_deprecated/fluid_preproc/fluid_test_computations/fluid_test_computations.cpp

```diff
@@ -220,8 +258,57 @@ ConvertDepthComputation::ConvertDepthComputation(test::Mat inMat, test::Mat outM
                                     cv::GMat out = InferenceEngine::gapi::ConvertDepth::on(in, depth);
                                     return cv::GComputation(cv::GIn(in), cv::GOut(out));
                                  }()
-                               , {to_own(inMat)}
-                               , {to_own(outMat)}
+                               , to_own(inMat)
+                               , to_own(outMat)
+                               })
+{}
+
+DivCComputation::DivCComputation(test::Mat inMat, test::Mat outMat, test::Scalar const& c)
+    : FluidComputation(new Priv{ []()-> cv::GComputation{
+                                    cv::GMat in;
+                                    cv::GScalar C;
+                                    cv::GMat out = in / C;
+                                    return cv::GComputation(cv::GIn(in, C), cv::GOut(out));
+                                }()
```

Comment:
Please add one more whitespace here to align with lambda's declaration's start indentation

Reply:
Done.